mods/persistence/modules/client/preferences.dm

```diff
@@ -0,0 +1,505 @@
+/* PLACEHOLDER VERB UNTIL SAVE INIT (or whatever the issue is) IS FIXED */
```

Comment:
You shouldn't copy the whole thing over if you're just overriding one or two procs. Its going to be a pain in the ass to maintain and fix bugs, and its just not good practice.